en/docs/fragments/register-a-spring-boot-app.md

```diff
@@ -1,55 +1,24 @@
-1. Access WSO2 Identity Server Developer Portal .
- 
- 2. Go to **Applications** and click **New Application**.
-  
- 3. Click **Show More**
- 
- 4. Open the  **OIDC Web Application** template.
-  
- 5. Enter a **Name** for the application 
-       
-    This indicates the name of your application.       
-    **Example**: sample-app
-   
- 6. Click **Next**
- 
- 7. Enter **Callback URL**. 
-      Add the flowwing two callBack URLs: 
+1.	Start WSO2 IS.
+
+2.	Access the Management Console (https://<IS_HOST\>:<PORT\>/carbon) to create a service provider.
+
+3.	Navigate to the **Service Providers** tab listed under the **Identity** section and click **Add**.
+
+4.	Provide the name for the service provider as `sample-app` and click **Register**. 
```

Comment:
shall we use the register-a-service-provider.md fragment here?

Reply:
We use different fragments for each app in samples since we need to give the sample values inline (For example, Provide the callback URL as `http://localhost:8080/spring-boot-app/login/oauth2/code/wso2`.)

Reply:
the aforementioned fragment doesn't contain info on callback URL

Reply:
Please refer step 7 and beyond of the register-a-spring-boot-app fragment.